data/darktableconfig.xml.in

```diff
@@ -832,6 +832,13 @@
     <shortdescription>select only new pictures</shortdescription>
     <longdescription>only select images that have not already been imported</longdescription>
   </dtconfig>
+  <dtconfig ui="yes">
```

Comment:
Why having a second option for this? I would have reused the above one? The details about filename, timestamp and whatever should not be exposed to user. I think dt should do its best to detect new images when possible.

So my question is can't we remove this and have a single controlling option?

Reply:
@quovadit claimed that the original label is not correct. I can agree it may be misleading if the user has not read the doc. 

>  The details about filename, timestamp and whatever should not be exposed to user. I think dt should do its best to detect new images when possible.

Agreed, but I think we have to explain this, in some way, to allow the user to understand what are the failing cases.
However I'm fine with the original text and we can give the explanation in the doc.
Let me know.

Reply:
Yes, please let's keep a single pref and have the doc enhanced. We really want to avoid too many prefs as we have seen that at some point it makes dt harder to setup and give an impression of "too many tweaks". Thanks.

Reply:
done